### PR TITLE
Added sensor strength to handle scripting the SeBo

### PR DESCRIPTION
**Pull request by Ebag333**

### CODE REVIEW

**Comment by Ebag333** [comment]
So this fixes the issue with scripting it....buuuuut......

Now it shows the ECM strength, because CCP decided to not make it the same.  So now it'll show 48% instead of 30%.

It'd be nice to do what we did with the RAH (similar to missiles, etc).  This was just a quick fix as long as I was in there doing the RAH stuff.

**Comment by Ebag333** [comment]
It was bugging me so rewrote the function.

http://puu.sh/poAX2/7b189ea3bb.png

Basically the same style function as RAH, missiles, etc. Much more clear to me, and based on feedback I've gotten, more clear to others.

**Comment by blitzmann** [comment]
Thanks!

File: gui/builtinViewColumns/misc.py [hunk 1]
```diff
@@ -231,21 +231,20 @@ def __getData(self, stuff):
...
-            if display == scanResBonus:
-                ttEntries.append("scan resolution")
-            tooltip = "{0} bonus".format(formatList(ttEntries)).capitalize()
+
+            text = "{0}% | {1}% | {2}%".format(
+                formatAmount(scanResBonus, 3, 0, 3),
+                formatAmount(lockRangeBonus, 3, 0, 3),
+                formatAmount(gravBonus, 3, 0, 3),
+            )
+            tooltip = "Applied bonuses:\n{0}% scan resolution| {1}% lock range | {2}% sensor strength".format(
```
Comment (Ebag333) [inline gui/builtinViewColumns/misc.py]: Missing space at: scan resolution|

:(

Comment (blitzmann) [inline gui/builtinViewColumns/misc.py]: I already fixed it in master =)

**Pull request merged**